pontoon/batch/utils.py

```diff
@@ -72,7 +117,6 @@ def find_and_replace(translations, find, replace, user):
         translations_to_create.append(translation)
 
     # Create new translations
-    changed_translations = Translation.objects.bulk_create(
-        translations_to_create
-    )
+    changed_translations = Translation.objects.bulk_create(translations_to_create)
```

Comment:
Iirc we've decided that we would go to 80 chars per line during out code-style meeting, didn't we? In which case, this is longer than 80. :)

Reply:
Ha! :)